src/components/data/PathListAutomation.js

```diff
@@ -0,0 +1,412 @@
+            errors.fileName = i18nCommon("required");
+        } else {
+            const invalidName = validateDiskResourceName(fileName, t);
+            if (invalidName) {
+                errors.fileName = invalidName;
+            }
+        }
+
+        if (!dest) {
+            errors.dest = i18nCommon("required");
+        }
+        return errors;
+    };
+
+    return (
+        <Formik
+            onSubmit={handlePathListCreation}
+            validate={validate}
+            initialValues={{ selectedPaths: [], dest: startingPath }}
+        >
+            {({ handleSubmit }) => {
+                return (
+                    <Form>
+                        {status === constants.LOADING && (
+                            <CircularProgress
+                                size={30}
+                                thickness={5}
+                                style={{
+                                    position: "absolute",
+                                    top: "50%",
+                                    left: "50%",
+                                }}
+                            />
+                        )}
+
+                        <Grid
+                            container
+                            direction="column"
+                            justify="center"
+                            alignItems="stretch"
+                            spacing={1}
+                        >
+                            {createPathListError && (
+                                <Grid item xs>
+                                    <ErrorTypographyWithDialog
+                                        baseId={baseId}
+                                        errorMessage={errorMsg}
+                                        errorObject={createPathListError}
+                                    />
+                                </Grid>
+                            )}
+                            <Grid item xs>
+                                <Typography variant="body2">
+                                    {t("pathListInputLbl")}
+                                </Typography>
+                                {infoTypeError && (
+                                    <ErrorTypographyWithDialog
+                                        baseId={baseId}
+                                        errorMessage={t("infoTypeFetchError")}
+                                        errorObject={infoTypeError}
+                                    />
+                                )}
+                                <Field
+                                    id={buildId(
+                                        baseId,
+                                        ids.PATH_LIST_AUTO_INPUTS
+                                    )}
+                                    name="selectedPaths"
+                                    required={true}
+                                    component={MultiInputSelector}
+                                    height="20vh"
```

Comment:
I noticed in the Chromium story that only 1 row at a time will display in the resource selector (at least in that view). So if the user selects more than 1 file or folder to process, they have to scroll to review all their selections. Maybe this could use an increased `height` to display more than 1 row at a time (maybe only an issue in smaller views):
https://www.chromatic.com/component?appId=5ebad5c6b991d60022b1e72e&name=Data+%2F+Path+List+Automation&buildNumber=1113

Reply:
Even in local dev mode on my laptop, I'm only seeing 1.5 rows:
![resource selector sizing in dev mode](https://user-images.githubusercontent.com/996408/101705318-e8547100-3a43-11eb-8869-b801b4d5c4e4.png)

Reply:
The reason I had to reduce height for Multi input selector and Info type selector is that, with increased height the dialog get into scrolling. When there is an error, it is displayed at the top of screen. It is easy to miss the error since hidden as users would scrolled down to click `Done` (happened even to me twice). I can look into it bit more.

Reply:
I made some adjustments to the `MultiInputSelector` table
![image](https://user-images.githubusercontent.com/1250790/101794807-4315c780-3b2d-11eb-8dd5-3095100fffae.png)

Also am displaying errors (if any) at the top and at the bottom to address the issue described above:
![image](https://user-images.githubusercontent.com/1250790/101794942-650f4a00-3b2d-11eb-909e-817ffc47952e.png)

Reply:
I also noticed in local dev on my laptop that this dialog would scroll, even with the short multi-input selector, and it also took me a while to find an error that was scrolled off the top.

So adding the error message by the cancel/done buttons is much better ✨